.github/workflows/go.yml

```diff
@@ -6,22 +6,47 @@ on:
   pull_request:
     branches: [ main ]
 
-jobs:
+env:
+  TZ: Asia/Shanghai
 
+jobs:
   build:
     name: Build
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-latest, ubuntu-latest]
     steps:
 
-    - name: Setup timezone
-      uses: szenius/set-timezone@v1.0
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
       with:
-        timezoneLinux: "Asia/Shanghai"
-        timezoneMacos: "Asia/Shanghai"
-        timezoneWindows: "China Standard Time"
+        go-version: ^1.13
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        # if [ -f Gopkg.toml ]; then
```

Comment:
any intention for these comments?

Reply:
These lines are copied from the previous code. I don't think it useful. Let me remove it?